react/src/lib/components/SubsurfaceViewer/components/Map.tsx

```diff
@@ -257,6 +257,12 @@ export interface MapProps {
 
     getTooltip?: TooltipCallback;
     cameraPosition?: ViewStateType;
+
+    /**
+     * Number of pixels used for the margin in matrix mode.
+     * Defaults to 0.
+     */
+    marginPixels?: number;
```

Comment:
Suggest nesting this prop in the `views` prop.

Reply:
I nested the "marginPixels" inside views as suggested.
However the "views as a callback" is probably better done in a separate PR as it is more of an architectural change.

Reply:
APIs have a tendency to cement quickly, so better to do it early, but I leave it for you to decide.